sdks/java/testing/load-tests/src/main/java/org/apache/beam/sdk/loadtests/SyntheticDataPubSubPublisher.java

```diff
@@ -50,9 +54,13 @@
  * <pre>
  *  ./gradlew :beam-sdks-java-load-tests:run -PloadTest.args='
  *    --insertionPipelineTopic=TOPIC_NAME
+ *    --kafkaBootstrapServerAddress=SERVER_ADDRESS
  *    --sourceOptions={"numRecords":1000,...}'
  *    -PloadTest.mainClass="org.apache.beam.sdk.loadtests.SyntheticDataPubSubPublisher"
  *  </pre>
+ *
+ * Parameter kafkaBootstrapServerAddress is optional. If provided, pipeline topic will be treated as
```

Comment:
I think we can do it even better. This pipeline could be designed in such a way that allows publishing to every sink type at the same time. So having: 

 - kafka related options & pubsub related options we can publish to both Kafka and Pubsub 
 - kafka related options: publish only to Kafka
 - pubsub related options: publish only to PubSub

WDYT?

Reply:
That's a good idea, implemented in 2nd commit